sysconfig/mac_osx/packages.yaml

```diff
@@ -0,0 +1,111 @@
+packages:
```

Comment:
This can come in handy in a future update of our spack : 
* https://spack.readthedocs.io/en/latest/build_settings.html

Reply:
https://github.com/spack/spack/pull/15158

Reply:
Not in our tree yet :(